examples/Readme.md

```diff
@@ -1,38 +1,57 @@
 
+## Models
+
+Classic models, some of which can be found in NetLogo's/MASON's example models.
+
+### bank_reserves
+A highly abstracted, simplified model of an economy, with only one type of agent and a single bank representing all banks in an economy.
+
 ### Color Patches
 A cellular automaton model where agents opinions are influenced by that of their neighbors. As the model evolves, color patches representing the prevailing opinion in a given area expand, contract, and sometimes disappear.
 
-### ConwaysGameOfLife
-
+### conways_game_of_life
 Implementation of [Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), a cellular automata where simple rules can give rise to complex patterns.
 
-### EpsteinCivilViolence
+### epstein_civil_violence
 Joshua Epstein's [model](http://www.uvm.edu/~pdodds/files/papers/others/2002/epstein2002a.pdf) of how a decentralized uprising can be suppressed or reach a critical mass of support.
 
-### Flockers
+### boid_flockers
 [Boids](https://en.wikipedia.org/wiki/Boids)-style flocking model, demonstrating the use of agents moving through a continuous space following direction vectors.
 
-### ForestFire
+### forest_fire
 Simple cellular automata of a fire spreading through a forest of cells on a grid, based on the NetLogo [Fire model](http://ccl.northwestern.edu/netlogo/models/Fire).
 
-### PD_Grid
+### hex_snowflake
+Conway's game of life on a hexagonal grid. TODO example/hex_snowflake/Readme.md needs to be updated.
+
+### pd_grid
 Grid-based demographic prisoner's dilemma model, demonstrating how simple mimicing can lead to the emergence of widespread cooperation -- and how a model activation regime can change its outcome.
 
-### Schelling (GUI and Text)
+### schelling (GUI and Text)
 Mesa implementation of the classic [Schelling segregation model](http://nifty.stanford.edu/2014/mccown-schelling-model-segregation/). 
 
-### Shape Example
-Example of grid display and direction showing agents in the form of arrow-head shape. 
-
-### Tutorial-Boltzmann_Wealth_Model
+### boltzmann_wealth_model
 Completed code to go along with the [tutorial]() on making a simple model of how a highly-skewed wealth distribution can emerge from simple rules.
 
-### WolfSheep
+### wolf_sheep
 Implementation of an ecological model of predation and reproduction, based on the NetLogo [Wolf Sheep Predation model](http://ccl.northwestern.edu/netlogo/models/WolfSheepPredation).
 
-### Sugarscape CG
+### sugarscape_cg
 Implementation of Sugarscape 2 Constant Growback model, based on the Netlogo
 [Sugarscape 2 Constant Growback](http://ccl.northwestern.edu/netlogo/models/Sugarscape2ConstantGrowback)
+
+### virus_on_network
+This model is based on the NetLogo model "Virus on Network".
+
+## Examples
```

Comment:
I think "examples" is too generic (the whole folder is called examples). Maybe "Feature examples"?